### PR TITLE
Updating workflows/computational-chemistry/gromacs-mmgbsa from 0.1.5 to 0.1.6 

### PR DESCRIPTION
**Pull request by gxydevbot**
Hello! This is an automated update of the following workflow: **workflows/computational-chemistry/gromacs-mmgbsa**. I created this PR because I think one or more of the component tools are out of date, i.e. there is a newer version available on the ToolShed.

By comparing with the latest versions available on the ToolShed, it seems the following tools are outdated:
* `toolshed.g2.bx.psu.edu/repos/chemteam/md_converter/md_converter/1.9.6+galaxy0` should be updated to `toolshed.g2.bx.psu.edu/repos/chemteam/md_converter/md_converter/1.9.7+galaxy0`

The workflow release number has been updated from 0.1.5 to 0.1.6.

### CODE REVIEW

File: workflows/computational-chemistry/gromacs-mmgbsa/gromacs-mmgbsa.ga [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "format-version": "0.1",
     "license": "MIT",
     "name": "MMGBSA calculations with GROMACS",
-    "release": "0.1.5",
+    "release": "0.1.24",
```
Comment (mvdbeek) [inline workflows/computational-chemistry/gromacs-mmgbsa/gromacs-mmgbsa.ga:14]:
```suggestion
    "release": "0.1.6",
```

File: workflows/computational-chemistry/gromacs-mmgbsa/gromacs-mmgbsa.ga [hunk 2]
```diff
@@ -828,7 +829,12 @@
                                 "output_name": "output"
                             }
                         },
-                        "inputs": [],
+                        "inputs": [
+                            {
+                                "description": "runtime parameter for tool AnteChamber",
+                                "name": "allparams"
+                            }
+                        ],
```
Comment (mvdbeek) [inline workflows/computational-chemistry/gromacs-mmgbsa/gromacs-mmgbsa.ga:837]:
```suggestion
                        "inputs": [],
```

File: workflows/computational-chemistry/gromacs-mmgbsa/gromacs-mmgbsa.ga [hunk 3]
```diff
@@ -1117,7 +1123,12 @@
                     "output_name": "outfile"
                 }
             },
-            "inputs": [],
+            "inputs": [
+                {
+                    "description": "runtime parameter for tool Split file",
+                    "name": "split_parms"
+                }
+            ],
```
Comment (mvdbeek) [inline workflows/computational-chemistry/gromacs-mmgbsa/gromacs-mmgbsa.ga:1131]:
```suggestion
            "inputs": [],
```

File: workflows/computational-chemistry/gromacs-mmgbsa/gromacs-mmgbsa.ga [hunk 4]
```diff
@@ -1334,7 +1345,16 @@
...
+            "inputs": [
+                {
+                    "description": "runtime parameter for tool Convert Parameters",
+                    "name": "param_inputs"
+                },
+                {
+                    "description": "runtime parameter for tool Convert Parameters",
+                    "name": "param_inputs"
+                }
+            ],
```
Comment (mvdbeek) [inline workflows/computational-chemistry/gromacs-mmgbsa/gromacs-mmgbsa.ga:1357]:
```suggestion
            "inputs": [],
```

File: workflows/computational-chemistry/gromacs-mmgbsa/gromacs-mmgbsa.ga [hunk 5]
```diff
@@ -1423,6 +1443,10 @@
                     "description": "runtime parameter for tool GROMACS simulation",
                     "name": "inps"
                 },
+                {
+                    "description": "runtime parameter for tool GROMACS simulation",
+                    "name": "inps"
+                },
```
Comment (mvdbeek) [inline workflows/computational-chemistry/gromacs-mmgbsa/gromacs-mmgbsa.ga:1449]:
```suggestion
```

File: workflows/computational-chemistry/gromacs-mmgbsa/gromacs-mmgbsa.ga [hunk 6]
```diff
@@ -1516,6 +1540,14 @@
...
             },
             "inputs": [
+                {
+                    "description": "runtime parameter for tool GROMACS simulation",
+                    "name": "inps"
+                },
+                {
+                    "description": "runtime parameter for tool GROMACS simulation",
+                    "name": "inps"
+                },
```
Comment (mvdbeek) [inline workflows/computational-chemistry/gromacs-mmgbsa/gromacs-mmgbsa.ga:1550]:
```suggestion
```

File: workflows/computational-chemistry/gromacs-mmgbsa/gromacs-mmgbsa.ga [hunk 7]
```diff
@@ -1605,6 +1637,14 @@
...
```
Comment (mvdbeek) [inline workflows/computational-chemistry/gromacs-mmgbsa/gromacs-mmgbsa.ga:1647]:
```suggestion
```

**Review by mvdbeek — COMMENTED** [review]
Drop inputs additions. Need to check out why Galaxy thinks these are disconnected ...

**Comment by lldelisle** [comment]
If someone look at this PR, would it be possible to incorporate:
https://github.com/galaxyproject/iwc/pull/530/files#diff-6cb346aaf5b88d9a286d77f4a0873f79b7e62441127cc22c09c6411a4cfcb68f

**Comment by lldelisle** [comment]
This is one of the workflows that autoupdate command raises an error. I guess someone needs to do it manually, a parameter may have changed names.

**Pull request closed without merging**